### ISSUE TITLE
Carousel documentation points to GH when it is available on amp.dev 

### ISSUE THREAD

**Issue by CrystalOnScript** [ampproject/amphtml#24419]
This should be updated to point to the website https://amp.dev/documentation/components/amp-carousel/?format=websites

**Comment by nainar** [comment]
@sparhami heads up
